### PR TITLE
Add Milesight-iot devices (was Ursalink)

### PR DESCRIPTION
**Pull request by Milesight-IoT**
Requested by: iot.support@milesight.com

Summary
Adds support for the Milesight-IoT AM107 sensor.

### CODE REVIEW

**Review by harryhua1989 — COMMENTED** [review]
all good.

File: vendor/index.yaml [hunk 1]
```diff
@@ -944,6 +944,10 @@ vendors:
     name: Boston Networks Limited
     vendorID: 598
 
+  - id: milesight-iot
+    name: Milesight IoT Co., Ltd.
+    vendorID: 600
```
Comment (johanstokking) [inline vendor/index.yaml:949]: We keep https://lora-alliance.org/wp-content/uploads/2020/11/LoRa_Alliance_Vendor_ID_for_QR_Code.pdf as authoritive index. 600 is reserved for future use.

Were you a LoRa Alliance member before, and was this your ID?

File: vendor/milesight-iot/profile-as923.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+vendorProfileID: 600
```
Comment (johanstokking) [inline vendor/milesight-iot/profile-as923.yaml:1]: The idea here is that each profile has its unique index that is issued by the vendor.

The `vendorID` + `vendorProfileID` are globally unique. I see now that you use 600 for different profiles.

**Review by johanstokking — CHANGES_REQUESTED** [review]

**Comment by harryhua1989** [comment]
Hello Johan,
Thanks for the comment

Our previous ID is 601, Ursalink Technology Co., Ltd., now we upgrade our brand to Milesight IoT Co., Ltd,  I suppose we can keep using 601 and using the new Company name.
If yes, I will publish the new document to github. 

Waiting for your reply.
Harry Hua
Field Application Engineer
Milesight IoT
4F, No.63-2, Wanghai Road, 2nd Software Park, Xiamen
361008, China
Tel: 86-592-5085280-6048
Ticket System: https://support.milesight-iot.com
Email: harry@milesight.com 
Skype: harry.milesight 
Mobile: (+86) 15260221327
        
 
From: Johan Stokking
Date: 2021-01-18 17:56
To: TheThingsNetwork/lorawan-devices
CC: harryhua1989; Comment
Subject: Re: [TheThingsNetwork/lorawan-devices] milesight-iot add new sensor AM107 (#69)
@johanstokking requested changes on this pull request.


In vendor/index.yaml:
> @@ -944,6 +944,10 @@ vendors:
     name: Boston Networks Limited
     vendorID: 598
 
+  - id: milesight-iot
+    name: Milesight IoT Co., Ltd.
+    vendorID: 600

We keep https://lora-alliance.org/wp-content/uploads/2020/11/LoRa_Alliance_Vendor_ID_for_QR_Code.pdf as authoritive index. 600 is reserved for future use.
Were you a LoRa Alliance member before, and was this your ID?


In vendor/milesight-iot/profile-as923.yaml:
> @@ -0,0 +1,8 @@
+vendorProfileID: 600

The idea here is that each profile has its unique index that is issued by the vendor.
The vendorID + vendorProfileID are globally unique. I see now that you use 600 for different profiles.

**Comment by johanstokking** [comment]
> Our previous ID is 601, Ursalink Technology Co., Ltd., now we upgrade our brand to Milesight IoT Co., Ltd, I suppose we can keep using 601 and using the new Company name. If yes, I will publish the new document to github.

Thanks. In that case, you can rename 601 to the new name (and change the folder name).

**Review by johanstokking — APPROVED** [review]
Thanks a lot

**Pull request merged**